tests/Unit/PointwiseFunctions/AnalyticSolutions/EinsteinSolutions/Test_KerrSchild.cpp

```diff
@@ -0,0 +1,187 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Index.hpp"
+#include "DataStructures/Tensor/EagerMath/Magnitude.hpp"
+#include "Domain/CoordinateMaps/AffineMap.hpp"
+#include "Domain/CoordinateMaps/CoordinateMap.hpp"
+#include "Domain/CoordinateMaps/ProductMaps.hpp"
+#include "Domain/LogicalCoordinates.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/EinsteinSolutions/KerrSchild.hpp"
+#include "tests/Unit/PointwiseFunctions/AnalyticSolutions/EinsteinSolutions/VerifyEinsteinSolution.hpp"
+#include "tests/Unit/TestHelpers.hpp"
+
+namespace {
+
+void test_schwarzschild() {
+  // Schwarzschild solution is:
+  // H        = M/r
+  // l_mu     = (1,x/r,y/r,z/r)
+  // lapse    = (1+2M/r){-1/2}
+  // d_lapse  = (1+2M/r){-3/2}(Mx^i/r^3)
+  // shift^i  = (2Mx^i/r^2) / lapse^2
+  // g_{ij}   = delta_{ij} + 2 M x_i x_j/r^3
+  // d_i H    = -Mx_i/r^3
+  // d_i l_j  = delta_{ij}/r - x^i x^j/r^3
+  // d_k g_ij = -2M x_i x_j x_k/r^5 + 4(M/r)x_(i/r [delta_{kj)}/r-x_k x_j)/r^3]
+  //          = -6M x_i x_j x_k/r^5 + 2 M x_i delta_{kj}/r^3
+  //                                + 2 M x_j delta_{ki}/r^3
+
+  // Parameters
+  //   ...for KerrSchild solution
+  const double mass = 1.01;
+  const std::array<double, 3> spin = {{0.0, 0.0, 0.0}};
+  const std::array<double, 3> center = {{0.0, 0.0, 0.0}};
+  //   ...for grid
+  const size_t grid_size = 3;
+  const std::array<double, 3> lower_bound = {{1.99, -0.01, -0.01}};
+  const std::array<double, 3> upper_bound = {{2.01, 0.01, 0.01}};
+
+  // Set up grid
+  const size_t data_size = pow<3>(grid_size);
+  Index<3> extents(grid_size);
+
+  // Set up map
+  CoordinateMaps::AffineMap x_map(-1., 1., lower_bound[0], upper_bound[0]);
+  CoordinateMaps::AffineMap y_map(-1., 1., lower_bound[1], upper_bound[1]);
```

Comment:
Can use using `AffineMap = CoordinateMaps::AffineMap;`
 and `using AffineMap3D =
      CoordinateMaps::ProductOf3Maps<AffineMap, AffineMap, AffineMap>`